src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
```

Comment:
don't be shy when giving your vars names 😉 
```suggestion
  const newObject = {};
```